community-membership.md

```diff
@@ -119,18 +118,18 @@ The following apply to the part of codebase for which one would be a reviewer in
 
 ### Responsibilities and privileges
 
-The following apply to the part of codebase for which one would be a reviewer in an
-[OWNERS](contributors/devel/owners.md) file (for repos using the bot).
+The following apply to the part of codebase for which one would be a reviewer in an [OWNERS] file
+(for repos using the bot).
 
 - Tests are automatically run for PullRequests from members of the Kubernetes GitHub organization
 - Code reviewer status may be a precondition to accepting large code contributions
-- Responsible for project quality control via [code reviews](contributors/devel/collab.md)
+- Responsible for project quality control via [code reviews]
   - Focus on code quality and correctness, including testing and factoring
   - May also review for more holistic issues, but not a requirement
-- Expected to be responsive to review requests as per [community expectations](contributors/devel/community-expectations.md)
+- Expected to be responsive to review requests as per [community expectations]
 - Assigned PRs to review related to area of expertise
 - Assigned test bugs related to area of expertise
-- Added to [`kubernetes-reviewers`](https://github.com/orgs/kubernetes/teams/kubernetes-reviewers)
+- Added to [kubernetes-reviewers]
 - Granted "read access" to kubernetes repo
 - Can champion incubator repos
```

Comment:
Incubator is dead, so we should remove that.

Reply:
Done